bin/crew

```diff
@@ -5,12 +5,10 @@ require_relative '../lib/color'
 abort 'Chromebrew should not be run as root.'.lightred if Process.uid == 0
 
 require 'find'
-require 'net/http'
 require 'uri'
 require 'digest/sha2'
 require 'json'
 require 'fileutils'
-require 'securerandom'
```

Comment:
Are you sure these modules are not needed?

Reply:
Yes, @saltedcoffii told me that in the previous pr

Reply:
Yes, securerandom is not needed.